CHANGELOG

```diff
@@ -1,3 +1,6 @@
+== Version 8.2.0
+- [Feature] Add support for Dynamic API Versioning ([#441](https://github.com/Shopify/shopify_python_api/pull/441))
```

Comment:
I think it would be worth expanding this, e.g.:

> Add support for Dynamic API Versioning. When the library is initialized, it will now make a request to Shopify to fetch a list of the available API versions.

(this may be useful information to know if someone is using [VCR.py](https://github.com/kevin1024/vcrpy) or similar in their test suite).

Reply:
I see! That makes sense!